bin/ebuild.sh

```diff
@@ -10,6 +10,10 @@ unalias -a
 unset BASH_COMPAT
 declare -F ___in_portage_iuse >/dev/null && export -n -f ___in_portage_iuse
 
+if [[ -v PORTAGE_EBUILD_EXTRA_SOURCE ]]; then
+	source "${PORTAGE_EBUILD_EXTRA_SOURCE}" || exit 1
+fi
+
```

Comment:
Since this variable doesn't need to stay in the environment, we could unset it here, or later in bin/save-ebuild-env.sh.

Reply:
Sure, sounds sensible.